src/mod/crc/obc_checksums.c

```diff
@@ -0,0 +1,122 @@
+#include <stdio.h>
```

Comment:
Ah,you copied this from hardwaretest repos...

My intention of crc was to get common usable code into:
ado-chip-175x-6x\ado_chip_175x_6x\src\ado_crc.c (*.h)

Not sure yet what you added used here but will comment it there.

Reply:
Yes, I copied from old repo ?

I can look through ado-chip and change to it if needed

Reply:
Checksum calculated with 
![image](https://user-images.githubusercontent.com/66519643/184124806-50975b1e-5b5e-4c0b-b718-62f06f92ae6f.png)

![image](https://user-images.githubusercontent.com/66519643/184124839-9355ee29-2ec1-4bef-bf8b-3917e6e76c7a.png)

is **wrong** . Therefore thruster did not worked.

My solution with commit :

https://github.com/RobertK66/obc_1769_core/commit/d5abfb96bfd8000728830127751fe0fd10528da7

Would be to copy paste correct checksum function (that works with thruster) into ado_crc.c


I dont have access rights for ado lib. Thus I will send updated files to your email.  Please  commit.

Reply:
I would sugest you move the crc_thruster() into thruster.c in order to make the code checked in compilable (merging to develop will generate a build and tries to make the image!).
I will move it to ado from there on later!